src/pg/test/e2e.test.js

```diff
@@ -0,0 +1,48 @@
+import { execFile as execFileCb } from 'child_process';
+import { promisify } from 'util';
+import { connOptions, populate } from './setup.js';
+
+const execFile = promisify(execFileCb);
+const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
+
+describe('pg-e2e', () => {
+  beforeAll(async () => {
+    try {
+      await execFile('docker', [
+        'run',
+        '--name',
+        'graffypg',
+        '-p',
+        `${connOptions.port}:5432`,
+        '-e',
+        `POSTGRES_PASSWORD=${connOptions.password}`,
+        '-d',
+        'postgres:alpine',
+      ]);
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(
+        'Could not start a test Postgres server using Docker.\n' +
```

Comment:
It works properly with docker. However, there is a challenge of testing it in github (docker in docker doesn't seem like a good idea). Looking at https://www.atdatabases.org/docs/pg-test please! I'm trying to simplify this open source (only the testing part). They are using CI for github.